pkg/META.in

```diff
@@ -10,7 +10,6 @@ archive(syntax, preprocessor, utf8) = "-ignore"
 preprocessor = "ocamlreason_preprocessor_where_does_this_end_up"
 
 # Toplevel
-archive(byte, toploop) = "@compiler-libs/ocamlcommon.cma @str/str.cma"
```

Comment:
Looks like rtop complains about missing dependencies with this change. If I add this line back as 
```
archive(byte, toploop) = "@compiler-libs/ocamlcommon.cma @re/re.cma @re/re_emacs.cma @re/re_str.cma"
```
then it fails to start entirely with:
```
Reason: Meta Language Utility
Fatal error: exception Syntax_util.Error(_, _)
```

But when I checkout @8e37c84bd which was right before the re changes it has the same fatal error, so I'm guessing rtop was broken before this PR.